README.md

```diff
@@ -15,6 +15,12 @@ Works in both browser & node environment.
 * JSON messaging system
 * Send & Respond to messages in a chain using Promise
 
+## MITM warnings
```

Comment:
Add this at the end of README, plus add a sub-section like "possible solutions", the most important part would be to emphasize "use open, public, trusted trackers" like https://openwebtorrent.com/